docs/source/copydbinprod.rst

```diff
@@ -0,0 +1,137 @@
+.. _copydbinprod:
+
+===========================================
+Running ``ghostferry-copydb`` in Production
+===========================================
+
+Assuming you have gone through :ref:`tutorialcopydb`, you probably want to run
+``ghostferry-copydb`` in production. The general workflow is relatively
+similar, with some differences. You should keep the tutorial as a starting
+point for your own playbook as most steps will largely be the same.
+
+Prerequisites
+-------------
+
+Before you start, you need to know if you can even use Ghostferry. Some points
+to consider about this are:
+
+- Ghostferry on its own does not enable 0 downtime moves. The downtime for the
+  app will be minimal compared to other methods but still non-zero.
+
+  - Figure out how much downtime you are willing to tolerate. Using Ghostferry,
+    one can realistically achieve downtime on the order of seconds.
+
+- The source database must be running with `FULL image`_ `ROW based replication`_.
+
+  - Without this, it is not possible to run Ghostferry safely.
+
+- Tables to be copied have integer primary keys.
+
+  - An issue exists to fix this limitation here:
+    `<https://github.com/Shopify/ghostferry/issues/2>`_.
+  - As a work around, you can use mysqldump during the cutover stage to migrate
+    those tables.
+
+- There are no foreign key constraints in your tables.
+
+  - You should remove these constraints before running Ghostferry.
+
+- ``ghostferry-copydb`` can only copy a whole table at a time.
+
+  - If you need copy a subset, use ghostferry as a library to build your own
+    application.
+
+- Ghostferry should only be run on the master database where writes occur.
```

Comment:
Could clarify a bit here what this means, like "In case of a multi-node replication setup, ..."